File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/AllyBaronKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class AllyBaronKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playAllyBaronKillAnimation() {
        new AnimationTester().testAnimation(new AllyBaronKillAnimation());
    }
}

File: test/hud/animations/AllyBaronKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {AllyBaronKillAnimation} from '../../../src/hud/animations/AllyBaronKillAnimation.js';

/**
 * Integration test for ally Baron kill animation on actual hardware.
 */
describe('AllyBaronKillAnimation Integration', () => {
    test('playAllyBaronKillAnimation', async () => {
        await new AnimationTester().testAnimation(new AllyBaronKillAnimation());
    }, 30000);
});
